RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # ChefDK 1.3 Release Notes
 
+## Note
+
+There is a known issue on the Windows platform with FIPS usage. If this would affect you, please continue to use ChefDK 1.2.22 until we resolve this issue with a patch release.
```

Comment:
Maybe `There is a known issue on the Windows platform that prevents FIPS usage` to be more clear? Pretty nit-picky though, it is good either way!